archetypes/Pools/PoolsTable/index.tsx

```diff
@@ -570,6 +571,9 @@ const TokenRows: React.FC<
     pastUpkeepTokenInfo,
     poolTicker,
 }) => {
+    const getImported = useStore(selectImportedPools);
+    const isImportedPool = useMemo(() => getImported.some((v) => v.address === poolAddress), [poolAddress]);
```

Comment:
`getImported` sounds like a function that you call to get imported pools, this could be renamed to just `importedPools`